apps/studio/components/interfaces/Reports/GridResize.tsx

```diff
@@ -1,6 +1,5 @@
 import RGL, { WidthProvider } from 'react-grid-layout'
 import 'react-grid-layout/css/styles.css'
-import 'react-resizable/css/styles.css'
```

Comment:
am removing this cause i keep running into this error somehow
![image](https://github.com/user-attachments/assets/4975662f-7495-4269-97a3-8353a2bbf7f3)

but i can verify that removing it doesnt cause any style loss or changes